packages/components/progress-bar/src/progress-bar.ts

```diff
@@ -0,0 +1,98 @@
+import { localized, msg } from '@lit/localize';
+import { type ScopedElementsMap, ScopedElementsMixin } from '@open-wc/scoped-elements/lit-element.js';
+import { Icon } from '@sl-design-system/icon';
+import { type CSSResultGroup, LitElement, type TemplateResult, html, nothing } from 'lit';
+import { property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import styles from './progress-bar.scss.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'sl-progress-bar': ProgressBar;
+  }
+}
+
+export type ProgressState = 'active' | 'success' | 'warning' | 'error';
+
+/**
+ * Progress bar component that can be used to communicate process status.
+ * Hidden aria-live element will help making the progress bar more accessible and should cause fewer bugs in products than leaving it up for the developers.
```

Comment:
this is not very nice towards the developers of the products ;)
Maybe it's nicer to say they don't need to worry about it than to say they'll probably mess it up :P

Reply:
Right 😆